src/pf-popover/pf-popover.component.js

```diff
@@ -0,0 +1,430 @@
+        this._showPopover();
+        //notify frameworks
+        this.dispatchEvent(new CustomEvent('pf-popover.opened', {}));
+      }
+    }, 20);
+  }
+
+  /**
+   * The popover close method
+   */
+  close() {
+    clearTimeout(this._timer);
+    this._timer = setTimeout(() => {
+      if (this.popover && this.popover !== null) {
+        pfUtil.removeClass(this.popover, 'in');
+        setTimeout(() => {
+          this._removePopover();
+          //notify frameworks
+          this.dispatchEvent(new CustomEvent('pf-popover.closed', {}));
+          // reset position after popover is closed
+          this._placement = this.getAttribute('placement') ? this.getAttribute('placement') : 'right';
+        }, this._duration);
+      }
+    }, this._delay + this._duration);
+  }
+
+  /**
+   * Removes the popover
+   * @private
+   */
+  _removePopover() {
+    this.popover && this._container.removeChild(this.popover);
+    this.popover = null;
+  }
+
+  /**
+   * Creates the popover
+   * @private
+   */
+  _createPopover() {
+    let clone = document.importNode(this._template.content, true);
+    let popoverInner = clone.querySelector('.popover-content');
+    let popovertitle = clone.querySelector('.popover-title');
+    let closeButton = document.createElement('template');
+    closeButton.innerHTML = `<button type="button" class="close"><span class="pficon pficon-close"></span></button>`;
+
+    if (this._popoverTitle === '' && !this._dismissible) {
+      popovertitle.parentNode.removeChild(popovertitle);
+    } else {
+      //set popover title
+      popovertitle.innerHTML = this._popoverTitle;
+
+      if (this._dismissible) {
+        popovertitle.appendChild(closeButton.content);
+      }
+    }
+    //set popover content
+    popoverInner.innerHTML = this.content;
+
+    //append to the container
+    this._container.appendChild(clone);
+
+    //set reference to appended node
+    this.popover = this._container.querySelectorAll('.popover:last-child')[0];
+    this.popover.style.display = 'block';
+    this.popover.setAttribute('class', `popover ${this._placement} ${this._animation}`);
+  }
+
+  /**
+   * update the placement of popover
+   */
+  _updatePlacement() {
+    switch (this._placement) {
+      case 'top':
+        return 'bottom';
+      case 'bottom':
+        return 'top';
+      case 'left':
+        return 'right';
+      case 'right':
+        return 'left';
+      default:
+        return this._placement;
+    }
+  }
+
+  /**
+   * Styles the popover based on placement attribute
+   * @private
+   */
+  _stylePopover() {
+    const rect = this._target.getBoundingClientRect(); //popover real dimensions
+
+    const // link rect | window vertical and horizontal scroll
+      scroll = pfUtil.getScroll();
+
+    const //link real dimensions
+      linkDimensions = { w: rect.right - rect.left, h: rect.bottom - rect.top };
+
+    const popoverDimensions = { w: this.popover.offsetWidth, h: this.popover.offsetHeight };
```

Comment:
Apologizes for the delay... it looks like this call after the fact is affecting the positioning when the window is scaled... I tested it with something like this
```javascript
  //const popoverDimensions = { w: this.popover.offsetWidth, h: this.popover.offsetHeight };
  const popoverDimensions = { w: this._ogWidth, h: this._ogHeight };
```
where ```this._ogWidth``` and ```this._ogHeight``` are populated when the popover is first created, or where I used the original dimensions of the popover here instead of the scaled versions... 

So what's happening is...
- the popovers have a scaling min-width instead of being fixed... when the browser is scaled the popover scales width-wise hence grabbing the dimensions after the fact means the dimensions are scaled and the position is off. 

Since this potentially doesn't just affect popovers, but tooltips as well, i'll catch up with @bleathem and @priley86 and look at simply opening an issue on this after the fact. 

You can try to fix it if you want, but really for now just look at squashing those commits and we'll try to get this merged